packages/accordion/accordion.scss

```diff
@@ -56,7 +56,7 @@
     }
 
     &__title-text {
-        @include body-paragraph;
+        @include body-paragraph--desktop;
```

Comment:
Desktop? Ville vi ikke vekk i fra det?

Reply:
Jo, men tok bare en direkte oversettelse av det som var (siden dette uansett bare er en refaktorering), så tar vi en search and replace når vi har byttet navnene